__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -719,6 +719,33 @@ var Foo = function Foo() {
 "
 `;
 
+exports[`babel-plugin-react-native-classname-to-dynamic-style Should support multiple classnames type in array styles objects: Should support multiple classnames type in array styles objects 1`] = `
+"
+const 
+  rd = Math.random(10),
+  style1 = styles.bar,
+  style2 = rd % 2 === 1 ? styles.bay : styles.baz,
+  Foo = () => <div className={[styles.foo, style1, style2].join(' ')}>Bar</div>
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+'use strict';
+
+var _reactNativeDynamicStyleProcessor = require('react-native-dynamic-style-processor');
+
+var rd = Math.random(10),
+    style1 = styles.bar,
+    style2 = rd % 2 === 1 ? styles.bay : styles.baz,
+    Foo = function Foo() {
+  return React.createElement(
+    'div',
+    { style: [_reactNativeDynamicStyleProcessor.process(styles).foo, style1, style2] },
```

Comment:
Thanks for the test, it really makes it easier to understand which code is throwing the error.

When I look at the result from the transform, it seems that it's not doing exactly what it should do. The code for `styles.bar`, `styles.bay` and `styles.baz` don't run through the `_reactNativeDynamicStyleProcessor.process` function, which is needed to support for example viewport units and other things.
```

Reply:
I wonder if there is a good way of following the variables and change the code to be like this:

```
var rd = Math.random(10),
    style1 = _reactNativeDynamicStyleProcessor.process(styles).bar,
    style2 = rd % 2 === 1 ? _reactNativeDynamicStyleProcessor.process(styles).bay : _reactNativeDynamicStyleProcessor.process(styles).baz,
    Foo = function Foo() {
  return React.createElement(
    'div',
    { style: [_reactNativeDynamicStyleProcessor.process(styles).foo, style1, style2] },
```